### PR TITLE
processor/remotetapprocessor: update name in README, cross-link remotetap extension, fix "port" config option

### PR DESCRIPTION
**Pull request by ringerc**
**Description:** Update component name and config snippet in remotetap processor docs

The remotetap ("remote tap") processor calls itself the "websocket" processor in its docs, and its config snippet didn't make the correct configuration clear. Fix.

**Link to tracking Issue:** N/A

**Testing:** N/A docs only change

**Documentation:** Add surrounding "processors" and "remotetap" clause to config to put it in context. Add "remotetap" name in README.

### CODE REVIEW

File: processor/remotetapprocessor/README.md [hunk 1]
```diff
@@ -11,8 +11,8 @@
 [alpha]: https://github.com/open-telemetry/opentelemetry-collector#alpha
 [contrib]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
 <!-- end autogenerated section -->
-The WebSocket processor, which can be positioned anywhere in a pipeline, allows
-data to pass through to the next component. Simultaneously, it makes a portion
+The Remote App processor a.k.a. WebSocket processor can be positioned anywhere in a pipeline.
```
Comment (atoulme) [inline processor/remotetapprocessor/README.md:14]: This is not a Remote App processor, but indeed a Remote Tap, as in the ability to tap into the flow.

**Review by atoulme — CHANGES_REQUESTED** [review]
remoteap -> remotetap

File: processor/remotetapprocessor/README.md (hunk 1 above)
Comment (ringerc) [reply processor/remotetapprocessor/README.md]: @atoulme Yep, silly mistake on my part. I've pushed a fix. I didn't mean to open the PR yet and was still checking it.

**Comment by ringerc** [comment]
@atoulme I'll need to handle the CRA, but this is updated now.

I wanted to add a "basic usage example" section to `processor/remotetapprocessor`'s README, but can't actually make it work at all. I can raise a GH issue for that and add it later, but it might be best to figure out a minimum working config here so I can put it in the README at the same time as the other updates.

**Comment by ringerc** [comment]
Based on https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/19633#issuecomment-1931193475 it looks like the `extension/remotetapextension` may be incomplete & non-working. TBD.

**Comment by ringerc** [comment]
Marked draft until I can determine whether either `processor/remotetapprocessor` and/or `extension/remotetapextension` is supposed to be usable in current state in `main`

**Pull request closed without merging**